feat: UIG-2454 - vl-proza-message - baseUrl attribuut

ksvlaanderen opened the pull request:
data-vl-base-url attribuut toegevoegd
storybook verbeteringen
cypress tests toegevoegd

Goldflow commented on libs/components/src/lib/proza-message/stories/vl-proza-message-preloader.stories.ts:
is het doenbaar ook een story te maken voor `baseUrl` (ofwel hier of bij proza-message) het is in de beschrijving voor mij wel duidelijk maar misschien is het goed dat er nog eens een letterlijk voorbeeld word gemaakt - mss geen aparte story maar gewoon een code snippet waarin

Goldflow replied:
ok specifieke documentatie werd toegevoegd - goed!

Goldflow commented on apps/storybook-e2e/src/e2e/components/proza-message/vl-proza-message.stories.cy.ts:
eventueel zou je hier ook nog een test kunnen toevoegen met (bv.) `cy.intercept()` wanneer de `baseUrl` ingesteld is, dat effectief een call word gedaan naar de verwachte url - maar dat is mss overkill

Goldflow commented on libs/components/src/lib/proza-message/stories/vl-proza-message-preloader.stories-doc.mdx:
```diff
@@ -0,0 +1,27 @@
+
+### Legacy Documentatie
+
+**Legacy Documentatie:** https://webcomponenten.omgeving.vlaanderen.be/doc/VlProzaMessagePreloader.html
```
mss toch hier ook link naar legacy demo van proza-message te zetten? (gezien die ook preloader vernoemt) - doe dit naar eigen inzicht

ksvlaanderen replied:
Done!

Goldflow reviewed (changes requested):
ziet er goed uit
enkele kleine opmerkingen ivm documentatie & suggestie voor cypress

ksvlaanderen committed:
feat: UIG-2454 - vl-proza-message - baseUrl attribuut

data-vl-base-url attribuut toegevoegd
storybook verbeteringen
cypress tests toegevoegd

kspeltix commented:
:tada: This PR is included in version 1.7.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.7.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
